Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -341,12 +353,56 @@ extension XPCServer {
         try XPCServiceServer._forThisXPCService()
     }
     
-    /// Creates a new anonymous server that only accepts connections from the same process it's running in.
-    internal static func makeAnonymous() -> XPCServer & NonBlockingServer {
+    /// Creates a new anonymous server that accepts requests from the same process it's running in.
+    ///
+    /// Only a client created from an anonymous server's endpoint can communicate with that server. Retrieve the ``NonBlockingServer/endpoint`` and
+    /// then create a client with it:
+    /// ```swift
+    /// let server = XPCServer.makeAnonymous()
+    /// let client = XPCClient.fromEndpoint(server.endpoint)
+    /// ```
+    ///
+    /// > Important: No requests will be processed until ``NonBlockingServer/start()`` or ``startAndBlock()`` is called.
+    ///
+    /// ## Usage
+    /// If you need this server to be communicated with by clients running in a different process, use ``makeAnonymous(clientRequirements:)`` instead.
```

Comment:
This isn't really "usage" information, it's kind of the opposite. Perhaps this should be an `Important`, `Attention`, `Note` or `Remark`?

Reply:
Hmm, yeah I was unsure of how to refer to this. I'll use a `Note`.